Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM debian:12-slim
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+# define copa_version arg with default value of 0.3.0
+ARG copa_version=0.3.0
```

Comment:
can we default to latest? by parsing from https://api.github.com/repos/project-copacetic/copacetic/releases/latest

Reply:
we can’t set a build time ARG dynamically when running the dockerfile, so I took the latest the the action file and passed that arg into the action container.